internal/buildscripts/packaging/fpm/common.sh

```diff
@@ -65,6 +66,17 @@ create_user_group() {
         sudo useradd --system --user-group --no-create-home --shell /sbin/nologin $SERVICE_USER
 }
 
+download_jmx_metric_gatherer() {
+    local version="$1"
+    local buildroot="$2"
+
+    JMX_METRIC_GATHERER_RELEASE_DL_URL="https://repo1.maven.org/maven2/io/opentelemetry/contrib/opentelemetry-jmx-metrics/$version/opentelemetry-jmx-metrics-$version.jar"
+    mkdir -p "$buildroot/opt"
+
+    echo "Downloading ${JMX_METRIC_GATHERER_RELEASE_DL_URL}"
+    curl -sL "$JMX_METRIC_GATHERER_RELEASE_DL_URL" -o "$buildroot/opt/opentelemetry-java-contrib-jmx-metrics.jar"
```

Comment:
Can you include the jar in `setup_files_and_permissions` below to ensure it is installed with the correct ownership/permissions?

Reply:
Good idea, I've added it.